src/com/lingoport/demo/java/ExampleMain.java

```diff
@@ -52,6 +52,7 @@ public static void main(String[] args){
         frame.pack();
         frame.setVisible(true);
         String strTitle = "Example Address Book -- " + locale;
+        String addedString = " This is new " + locale;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) active Embedded Strings : String " This is new " [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=gyzr%3AGlobalyzerESConcatCheck)